simulariumio/trajectory_converter.py

```diff
@@ -148,6 +149,35 @@ def get_subpoints_xyz(subpoints: np.array, n_subpoints: np.array) -> np.array:
                 )
         return xyz_subpoints.reshape(1, -1, 3)
 
+    @staticmethod
+    def translate_positions(data: AgentData, translation: np.ndarray) -> AgentData:
+        total_steps = data.times.size
+        max_subpoints = int(np.amax(data.n_subpoints))
+        for time_index in range(total_steps):
+            for agent_index in range(int(data.n_agents[time_index])):
+                display_type = data.display_type_for_agent(time_index, agent_index)
+                translate_subpoints = (
+                    max_subpoints > 0 and display_type != DISPLAY_TYPE.SPHERE_GROUP
+                )
```

Comment:
is this just checking if the agent has subpoints? If so, I think it would be better to name `translate_subpoints` something like `has_subpoints` because at first glance I thought this was an option and I was confused why you would sometimes not want to translate the subpoints, and like 177-178 is pretty confusing